src/roles/common/templates/group_vars.all.j2

```diff
@@ -9,6 +9,11 @@ user_ssh_pub_key: {{ common.user_ssh_pub_key | default('~/.ssh/id_rsa.pub') }}
 xmpp_tls: {{ common.xmpp_tls | default(True) }}
 openflow_tls: {{ common.openflow_tls | default(True) }}
 
+stats_out: {{ common.stats_out | default('false') }}
+{% if common.stats_out | default(false) %}
+stats_out_proxy: {{ common.stats_out_proxy | default(nuhs[0].hostname) }}
```

Comment:
Will this default error out if there is no NUH.

Reply:
Really good eye noticing this.  But, it can't happen.  I have a check in read-deployment (before inventory is generated):

```
  - name: Check for valid stats-out proxy address
    assert:
      that: "common.stats_out_proxy is defined or nuhs | default([]) | length > 0"
      msg: "Missing stats_out_proxy address in common.yml and no NUHs are defined to be used as a proxy."

  when: common.stats_out | default(false)
```